src/transforms/serializer.js

```diff
@@ -2,7 +2,7 @@
 
 const ProtoDef = require('protodef').ProtoDef
 const Serializer = require('protodef').Serializer
-const Parser = require('protodef').FullPacketParser
+const Parser = require('protodef').LazyPacketParser
```

Comment:
use the FullPacketParser for 1.7 (you can check how the compiled variable is defined)